test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -39,6 +40,9 @@ const (
 	threshold60K = 30
 )
 
+// TODO: Need a way to eliminate these.
+var nodes, pods int
```

Comment:
yeah lets find a way to embed these into a struct somwhere so the user can specify pods/nodes in the same place they specify predicate/priority details.